### PR TITLE
Interactive_Colab_Notebook_for_predicting_IR_and_Raman_Bands.ipynb

### PR DESCRIPTION
**Pull request by O-Mary**
Here

### CODE REVIEW

File: Interactive_Colab_Notebook_for_predicting_IR_and_Raman_Bands.ipynb [hunk 1]
```diff
@@ -4,7 +4,10 @@
   "metadata": {
```
Comment (ignaciomigliaro) [inline Interactive_Colab_Notebook_for_predicting_IR_and_Raman_Bands.ipynb:4]: <p>I don't like how the moving text looks, it makes it look very unproffesional. If we can remove this . </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ignaciomigliaro/Group_theory_for_IR/pull/3/discussion/#comment-1452865373'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Comment (ignaciomigliaro) [inline Interactive_Colab_Notebook_for_predicting_IR_and_Raman_Bands.ipynb:4]: <p>Same here remove this. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ignaciomigliaro/Group_theory_for_IR/pull/3/discussion/#comment-1452865374'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Comment (ignaciomigliaro) [inline Interactive_Colab_Notebook_for_predicting_IR_and_Raman_Bands.ipynb:4]: <p>Again not necessary to have moving text. </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ignaciomigliaro/Group_theory_for_IR/pull/3/discussion/#comment-1452865375'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Comment (ignaciomigliaro) [inline Interactive_Colab_Notebook_for_predicting_IR_and_Raman_Bands.ipynb:4]: <pre class="ql-syntax" spellcheck="false">Line #3.    Dont rush! take time to read the text and then run the next five code cells only one time each
</pre><p>Remove moving text</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ignaciomigliaro/Group_theory_for_IR/pull/3/discussion/#comment-1452865377'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='21'/>

Comment (O-Mary) [reply Interactive_Colab_Notebook_for_predicting_IR_and_Raman_Bands.ipynb:4]: Sure we can...

**Pull request merged**